shared/react-native/ios/Keybase/AppDelegate.m

```diff
@@ -126,4 +126,20 @@ - (void)application:(UIApplication *)application didReceiveLocalNotification:(UI
   [RCTPushNotificationManager didReceiveLocalNotification:notification];
 }
 
+- (void)applicationDidEnterBackground:(UIApplication *)application
+{
+  UIImageView *imageView = [[UIImageView alloc] initWithFrame:self.window.bounds];
+  imageView.contentMode = UIViewContentModeCenter;
+  imageView.tag = 42;    // Give some decent tagvalue or keep a reference of imageView in self
```

Comment:
I would keep access to imageView instead of using tag.

Then you can do `[imageView removeFromSuperView]` instead of trying to find it.